### ISSUE TITLE
v0.9 Scrolling inside tmux not working as it's used to be

### ISSUE THREAD

**Issue by feryardiant** [mskyaxl/wsl-terminal#183]
## How to reproduce

- enter tmux
- make sure to enable mouse interaction `setw -g mouse on` in `.tmux.conf`
- hit enter multiple times, or open any `man` pages
- scroll using touchpad or mouse scroll wheel

## Expected

The text on screen is scrolled as it should be in the previous versions

## Actual

It's scrolling command history before actually scrolling text on screen

---

Tried to roll back to v0.3.13 everything's still works fine
Open any file using vim inside tmux, srolling still works fine
Tried compared to Windows Terminal, [video here](https://share.getcloudapp.com/6qu50v1K)

**Comment by mskyaxl** [comment]
Thanks for the detailed report! I will have a look :)

**Comment by mskyaxl** [comment]
Hi @feryardiant,

I am not able to reproduce or I don't understand the issue completely.

![image](https://user-images.githubusercontent.com/5708771/79738621-44278980-82fd-11ea-9f10-a95ec02c3771.png)

How do you start the wsl-terminal? with login?

**Comment by feryardiant** [comment]
Hi @mskyaxl 

Sorry for late reply, here's my `wsl-terminal` shortcut' property:

```
C:\tools\lib\wsl-terminal\bin\mintty.exe --wsl --rootfs=// --configdir "C:\tools\lib\wsl-terminal\etc" -i "C:\tools\lib\wsl-terminal\open-wsl.exe"  -t "        " -e /bin/wslbridge2  -e SHELL="/usr/bin/zsh"
```

and yes I am on [login shell](https://github.com/feryardiant/dotfiles/blob/master/.bashrc#L5-L7)

**Comment by feryardiant** [comment]
Anyway, for some reasons it might be there's some issues with my [dotfiles config](https://github.com/feryardiant/dotfiles) but I still don't know. I'll do some investigate & let you know once I figured it out.

**Comment by mskyaxl** [comment]
> 
> 
> Hi @mskyaxl
> 
> Sorry for late reply, here's my `wsl-terminal` shortcut' property:
> 
> ```
> C:\tools\lib\wsl-terminal\bin\mintty.exe --wsl --rootfs=// --configdir "C:\tools\lib\wsl-terminal\etc" -i "C:\tools\lib\wsl-terminal\open-wsl.exe"  -t "        " -e /bin/wslbridge2  -e SHELL="/usr/bin/zsh"
> ```
> 
> and yes I am on [login shell](https://github.com/feryardiant/dotfiles/blob/master/.bashrc#L5-L7)

seems to be correct
can you try to run ```<your path>\wsl-terminal\open-wsl.exe -l```?

**Comment by feryardiant** [comment]
> if zsh is your shell `.bashrc` will not be sourced at startup

Yes, you're right and I have to admit that I don't really familiar in this regard, lol. But this answer helps me to verify 

![image](https://user-images.githubusercontent.com/508665/80090022-13f41b00-8589-11ea-8167-949b7733f170.png)

> can you try to run `<your path>\wsl-terminal\open-wsl.exe -l`?

I have tried it and it give me these

**Comment by mskyaxl** [comment]
```C:\tools\lib\wsl-terminal\bin\mintty.exe --wsl --rootfs=// --configdir "C:\tools\lib\wsl-terminal\etc" -i "C:\tools\lib\wsl-terminal\open-wsl.exe -l"  -t "        " -e /bin/wslbridge2  -e SHELL="/usr/bin/zsh"```

this cannot work because -i argument for mintty is the selection of the icon, try please:
```C:\tools\lib\wsl-terminal\bin\open-wsl.exe -l``` and that's all
